exercises/pov/canonical-data.json

```diff
@@ -0,0 +1,348 @@
+{
+  "from_pov": {
+    "description": [
+      "Reroot a tree so that its root is the specified node.",
+      "In this way, the tree is presented from the point of view of the specified node.",
+      "The input trees used here are those in the `trees` section of this file.",
```

Comment:
I chose to the put the input trees in the `"trees"` section because they are used more than once and it seemed useful to allow people to easily see that they are the same six trees being reused.

In contrast, I put the expected output trees inline since they are only used once.